.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -0,0 +1,72 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+
+  # MacOS specific build settings
+  CIBW_BEFORE_ALL_MACOS: |
+    brew uninstall --force oclint
+    brew install gcc libomp
+
+  # Python build settings
+  CIBW_BEFORE_BUILD: |
+    pip install pybind11
+
+  # Testing of built wheels
+  CIBW_TEST_REQUIRES: numpy scipy pytest pytest-cov pytest-mock flaky
+
+  # Note: breaking up tests so that no test cases hang on Windows, to be reverted
+  CIBW_TEST_COMMAND: |
+    pip install numpy==1.19.5 scipy
+    pl-device-test --device=lightning.qubit --skip-ops -x --tb=short --no-flaky-report
+
+  # Skip PPC tests due to lack of numpy/scipy wheel support
+  CIBW_TEST_SKIP: "*-manylinux_{aarch64}"
+
+  # Use Centos 7 wheel-builder for ARM, Debian for PPC
+  CIBW_MANYLINUX_AARCH64_IMAGE: manylinux2014
+
+jobs:
+  linux-wheels-aarch64:
+    if: github.ref == 'refs/heads/master'
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        arch: [aarch64]
+    name: ubuntu-latest::aarch64
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.4.1
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
+
+      - uses: docker/setup-qemu-action@v1
+        name: Set up QEMU
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel==1.12.0
```

Comment:
Is it best to version pin?

Reply:
We had an older version pinned, so I updated to this, but I agree. I don't see any reason to do so.

Reply:
Actually, on second thought, I think this is fine for now, as it provides all the functionality we currently need.

Reply:
Yes, because `cibuildwheel` updates often, changing behaviour and environment variable names. We previously did not pin, and would constantly have to fix the CI.

Having said that, it is important to make sure that we are updating the version of `cibuildwheel` as often as possible, because they are frequently fixing bugs and making sure that all build options and CI services work correctly